rumqttc/src/lib.rs

```diff
@@ -432,6 +457,7 @@ impl MqttOptions {
             last_will: None,
             conn_timeout: 5,
             manual_acks: false,
+            network_option: NetworkOptions::new(),
```

Comment:
Move `conn_timetout` to `NetworkOptions` and move `NetworkOptions` as an argument to `Eventloop`